package.json

```diff
@@ -156,6 +156,7 @@
     "node-polyglot": "^2.4.0",
     "os-browserify": "^0.3.0",
     "p-ratelimit": "^1.0.1",
+    "pretty-ms": "^8.0.0",
```

Comment:
Reckon `date-fns` should be able to display things in a pretty way as well but happy with this one as a simpler option

Reply:
One thing to note is this package won't localize date formats, if we care about that

Reply:
Yeah reading this again we should be fine, some languages may use different format but that's most definitely edge case, and I doubt we would ever have such a bigly trade timed in days as well:
<img width="438" alt="image" src="https://github.com/shapeshift/web/assets/17035424/6227f16c-4e1d-49d1-8d77-ed280f573978">